portfolio/src/main/webapp/script.js

```diff
@@ -71,7 +68,7 @@ async function heyBilly() {
 
 // Not in use, just for practice and reference
 function heyBillyArrow() {
-  fetch('/data')     .then(response => {
+  fetch('/data').then(response => {
     response.text()}).then((quote)  => {
```

Comment:
There should be a semicolon after "response.text();" and I recommend putting the rest of this line on another line.